c10/core/ScalarType.h

```diff
@@ -425,6 +479,8 @@ static inline ScalarType promoteTypes(ScalarType a, ScalarType b) {
   constexpr auto c8 = ScalarType::ComplexDouble;
   constexpr auto b1 = ScalarType::Bool;
   constexpr auto bf = ScalarType::BFloat16;
+  constexpr auto b8 = ScalarType::Float8_e5m2;
+  constexpr auto h8 = ScalarType::Float8_e4m3;
```

Comment:
b8/h8 are funny names to call these lol.

Reply:
Are there funny enough to be changed, or can I leave there?